deployment/shell.nix

```diff
@@ -87,8 +91,9 @@ let
           echo "[deploy-nix]: machines.json is not present. Aborting."
           exit 1
         fi
+
         echo "[deploy-nix]: copying machines.json .."
-        cp ./machines.json ./morph
+        cat ./machines.json | jq --arg rev ${rev} '. + {rev: $rev}' > ./morph/machines.json
```

Comment:
This seems a little dodgy, but probably works. Isn't there some kind of notion of variable in the tooling that we should be using?

Reply:
I agree this isn't great, but there is no concept of passing arguments to morph deployments.

I think one thing that makes it look even more dodgy than it is, is the misleading name `machines.json` - this really ought to be renamed to `deployment.json` as it provides various deployment relevant information, the git revision being one of them.

I could instead write to a file `git-revision.json` and read that but I think i would prefer having _one_ input file to morph instead of multiple separate ones.